setup.py

```diff
@@ -42,7 +48,28 @@
 printer = print  # noqa: T001
 
 
+class SetupClean(clean):
+
+    def run(self):
+        clean.run(self)
+        for filename in glob.glob('bibliopixel/*.pyc'):
```

Comment:
There shouldn't be _any_ such files, should there??  All the .pyc files live in `__pycache__`.

Reply:
No, but in certain development situations, you may find yourself with random cached code that somehow makes it into a git commit.